scripts/publish-docker.sh

```diff
@@ -19,10 +19,12 @@ underscore_name="$(echo -n "$log_name" | tr - _)"
 docker push "$base_tag-$hash"
 docker push "$base_tag-$version"
 
-mkdir -p "$artifacts"
+if [ -z "$artifacts" ]; then
```

Comment:
this if statement looks good but I think you'll have to change the `if [ "$#" -ne 5 ]` clause uptop

Reply:
Oohh good catch